### PR TITLE
Added Arabic translations

### PR DESCRIPTION
**Pull request by YDA93**

### CODE REVIEW

File: captcha/locale/ar/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Django-recapcha\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-12-26 08:32-0500\n"
+"PO-Revision-Date: 2014-09-15 16:47-0300\n"
+"Last-Translator: Rafael Ponieman <rafadev@gmail.com>\n"
+"Language-Team: \n"
+"Language: Spanish\n"
```
Comment (Stormheg) [inline captcha/locale/ar/LC_MESSAGES/django.po:14]: **Question:** are you sure this is correct? Shouldn't this be Arabic?

File: captcha/locale/ar/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
```
Comment (Stormheg) [inline captcha/locale/ar/LC_MESSAGES/django.po:18]: **Question:** are you sure these plural forms are correct for Arabic? [This source suggests there exist 6 plural forms in Arabic and gives a different formula](https://wiki.arabeyes.org/Plural_Forms).

**Review by Stormheg — COMMENTED** [review]
Hey @YDA93, thanks for the contribution. Two minor questions.

I see you also altered the compiled Indonesian and Japanese translations. Could you please drop those changes from your commit as they are not relevant to this PR? Thank you!

**Comment by YDA93** [comment]
@Stormheg  Done. Thanks.

**Review by Stormheg — APPROVED** [review]
Thanks!

**Pull request merged**